apps/web/src/providers/app-state-provider/Provider.tsx

```diff
@@ -33,7 +33,9 @@ export const AppStateProvider = ({
         await createDefaultWorkspace(dataCenter);
       }
       setUser(
-        (await dataCenter.getUserInfo(dataCenter.providers[0]?.id)) || null
+        (await dataCenter.getUserInfo(
+          dataCenter.providers.filter(p => p.id !== 'local')[0]?.id
+        )) || null
```

Comment:
Right now `dataCenter.providers` field has `local`(`tauri-ipc` in desktop app) and `affine` in order.
However the `local` provider does not have valid user info and shall be excluded.